### PR TITLE
Make player ghost armor transparent

### PR DESCRIPTION
**Pull request by Incompleteusern**
Yet untested
Extension of #636

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -97,4 +97,3 @@
 * Update qodana ([b6cf6e7](https://github.com/Wynntils/Artemis/commit/b6cf6e74231aeaaddae9daaeb885ce0052fcf210))
 
 ## 0.0.0 (2021-12-09)
-
```
Comment (DevScyu) [inline CHANGELOG.md:129]: You can revert this change, it is now ignored by spotless

Comment (kristofbolyai) [reply CHANGELOG.md:129]: its fine to leave it for now
